tensorflow_model_optimization/g3doc/guide/clustering/clustering_with_keras.ipynb

```diff
@@ -0,0 +1,1164 @@
+{
```

Comment:
<p>General principle: don't include implementation details in these highest level user-facing docs. </p><p> </p><p>Stripping and "clustering wrappers" is an implementation detail. The headers I propose above address this issue.</p><p> </p><p>I understand this is where our pruning docs are today.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/model-optimization/pull/330/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>Done, thanks.</p>